CHANGELOG.md

```diff
@@ -1,3 +1,64 @@
+v1.6.0
+------
+* **Release date:** 2018-12-21.
+* **Requires:** [tileserver v2.2.0](https://github.com/tilezen/tileserver/releases/tag/v2.2.0) and [tilequeue v2.2.0](https://github.com/tilezen/tilequeue/releases/tag/v2.2.0) and [rawr_tiles v1.0.0](https://github.com/tilezen/raw_tiles/releases/tag/v1.0.0) and [coanacatl v1.0.0](https://github.com/tilezen/coanacatl/releases/tag/v1.0.0).
+
+  #### ENHANCEMENTS
+
+  * **boundaries**: Strip long names from boundaries. [Issue #1683](https://github.com/tilezen/vector-datasource/issues/1683).
+  * **boundaries**: Remove id, id:left and id:right properties at mid and low zooms (z<13) to reduce file size. [Issue #1715](https://github.com/tilezen/vector-datasource/issues/1715).
+  * **boundaries**: Push locality boundary features down to min zoom 11. [Issue #1715](https://github.com/tilezen/vector-datasource/issues/1715).
+  * **boundaries**: Double simplification tolerance boundaries layers. [Issue #641](https://github.com/tilezen/vector-datasource/issues/641) and [PR #1718](https://github.com/tilezen/vector-datasource/pull/1718).
+  * **buildings:** Improve building merging and remove some mid-zoom content. Issues [#1686](https://github.com/tilezen/vector-datasource/issues/1686) and [#1732](https://github.com/tilezen/vector-datasource/issues/1732) [PR #1689](https://github.com/tilezen/vector-datasource/pull/1689), [#1704](https://github.com/tilezen/vector-datasource/pull/1704), and [PR #1739](https://github.com/tilezen/vector-datasource/pull/1739)
+  * **landuse**: Add `allotments` (community gardens), was already in POIs layer. #1742  
```

Comment:
You linked the PR manually (i.e: with `[PR ...](...)` syntax) in other lines. Github probably auto-links `#1742`, but perhaps use the plain Markdown syntax for consistency?